src/rapid_response_xblock/setup.cfg

```diff
@@ -0,0 +1,30 @@
+[tool:pytest]
```

Comment:
I'm not sure if we should keep this. I know we have setup.cfg in sysadmin too but I want to know what happens if we remove it? I only see that it's being used for `Flake8` config in this project

Reply:
The integration tests for each project (sys_admin and rapid_response) [run](https://github.com/mitodl/open-edx-plugins/blob/main/run_devstack_integration_tests.sh#L48-L49) from inside their respective project directories. When writing the GitHub workflow for this project, I attempted to run the tests from the root directory by configuring setup.cfg in the root directory. However, the tests failed to execute due to several different errors. I believe it's better to run them from their respective project folders and for this we require setup.cfg in each project separately.
We can however remove the [setup.cfg](https://github.com/mitodl/open-edx-plugins/blob/main/setup.cfg) file at the root level which is only being used for Flake8 configuration.